### PR TITLE
style + fix: move all consts to one file + disable console instrumentation in debug mode

### PR DESCRIPTION
**Pull request by wrn14897**

### CODE REVIEW

File: packages/node-opentelemetry/src/otel.ts [hunk 1]
```diff
@@ -32,37 +39,46 @@ export type SDKConfig = {
...
     console.warn(`${LOG_PREFIX} HYPERDX_API_KEY is not set`);
   }
 
-  const stopOnTerminationSignals = config.stopOnTerminationSignals ?? true; // Stop by default
-
   hdx('Initializing OpenTelemetry SDK');
-  const consoleInstrumentationEnabled = config.consoleCapture ?? true;
+  let consoleInstrumentationEnabled = config.consoleCapture ?? true;
+  if (DEFAULT_OTEL_LOG_LEVEL === DiagLogLevel.DEBUG) {
+    // FIXME: better to disable console instrumentation if otel log is enabled
```
Comment (colehpage) [inline packages/node-opentelemetry/src/otel.ts:70]: fine to leave until later?

File: packages/node-opentelemetry/src/otel.ts [hunk 2]
```diff
@@ -109,21 +129,21 @@ export const initSDK = (config: SDKConfig) => {
     ],
   });
 
-  if (env.OTEL_EXPORTER_OTLP_ENDPOINT && env.OTEL_EXPORTER_OTLP_HEADERS) {
+  if (env.OTEL_EXPORTER_OTLP_HEADERS || env.HYPERDX_API_KEY) {
```
[the remark this answers is not in the document]
Comment (wrn14897) [reply packages/node-opentelemetry/src/otel.ts:132]: Good call! Let me update the warning message. The idea is to start the sdk only when auth key is detected (either `HYPERDX_API_KEY` or `OTEL_EXPORTER_OTLP_HEADERS`)

File: packages/node-opentelemetry/src/otel.ts (hunk 1 above)
Comment (wrn14897) [reply packages/node-opentelemetry/src/otel.ts:70]: yeah its fine. currently we disable console instrumentation only if otel level is debug (which caused a recursion issue)

**Review by colehpage — APPROVED** [review]
🚢 🚢

**Pull request merged**